### ISSUE TITLE
Is this project active?

### ISSUE THREAD

**Issue by fresheneesz** [EqualExperts/Tayra#10]
The youtube video has been removed, and the http://www.eelabs.co.uk/ website is gone, meaning the downloads section is gone too.

**Comment by smeijer** [comment]
I guess that goes both ways. I was looking for a way to backup mongo by watching it's oplog. It was [this article](http://www.jroller.com/DhavalDalal/entry/tayra_an_incremental_backup_tool) that lead me to Tayra. But once I discovered the project, the information stream just stopped. I can't find any information on the net, about how it works, how to set it up, and how to use it. What the best strategies are to apply, if I need to use cronjobs of that "it just works". Do I need a separate server, or...

I found some slideshares, but to be honest, that's just not enough.

I'm pretty sure the project can be very useful, but without information people just don't start trying it out. Some (good) documentation, and demo video's on youtube for example, can really safe this project.

Even if it is just a "how to install on ubuntu 14.04" tutorial, followed up by a "how to restore data in case of..." tutorial.

Just my 2 cents.

**Comment by krishnan-mani** [comment]
@fresheneesz, your suggestions and requests are welcome and we'll respond shortly with more HOW-TO documentation that will helpfully set you up and running on your way using Tayra

**Comment by fresheneesz** [comment]
I agree with smeijer, we need something to go on if you want feedback. My feedback: write some documentation on how to run and use Tayra.

**Comment by esorek** [comment]
I would be interested to try it also, with good docs and if this project will be active.
It seems like a very useful tool.
Any plans for GA ?

**Comment by viveksd87** [comment]
Hi Guys,
Thank you for your feedback. Really appreciate the growing interest in Tayra.
However, have you been able to check this link for documentation?

http://htmlpreview.github.io/?https://github.com/EqualExperts/Tayra/blob/master/acceptance-tests/index.html

We have tried our best to provide with a documentation for any new interested person to get started with Tayra.
However, if you feel it's not up to the mark, please let us know how you would like it to be.

Thanks,
Vivek

**Comment by fresheneesz** [comment]
This is definitely a huge step in the right direction as compared to the deadlinks I was seeing before. I will have to check this out sometime this month. Incremental backup and restore seems critical in this day and age.

Its still somewhat vague to me how to actually do incremental backup and restore. Because the documentation is essentially organized as a list of examples, its very difficult to know everything Tayra can do. It would be much better if the documentation described the commands something like this:

**backup** - Copies the mongo server's oplog into a given set of backup files 

Parameters:
* **-s**: server location (eg. localhost)
* **--port** or **-p**: server port
* **-f**: file path where the backup will be stored
* **-t**: Listens to the oplog and backs up operations as they are written to the database. Without this parameter, the backup will quit once all the operations in the oplog are copied, and will not listen for new ones. *(Is this correct?)*
* **--fMax**: Maximum number of files to backup into *(how does this related to the -f parameter? what happens when the maximum number of files is reached? More info is needed here)*
* **--fSize**: Maximum file size of a backup file. The argument for this parameter should be something like "4000" for 4 kilobytes or "4MB" for 4 megabytes. (Is this correct? The description in the docs makes it seems like this actually relates to the total size of the backup across all backup files)

**restore** - replays the given backup oplog on the given mongo server

Parameters:
* **-d**: destination server *(I'd recommend making this parameter consistent with the corresponding parameter for *backup*, namely **-s**)*
* **--port**: The destination server port
* **-f**: The backup file to replay
* **-fAll**: *(What does this mean?)*
* **--sUntil**: The time to replay the oplog until. *(What format is this value in? Need more information about how to determine what value to use here)*

**Comment by viveksd87** [comment]
Ok. I think I understand your point. Although these one line descriptions are provided on the command line, there is a need for proper detailed description of the commands and their parameters.
We can definitely work on it, and for starters make it a part of the readme.

Thank you for the inputs @fresheneesz .

**Comment by cu12** [comment]
+1 for more documentation, I think the tool would be very useful with proper documentation.

**Comment by spethe** [comment]
Hi cu12,
Firstly, Its good to know that the tool would be useful. We will certainly attempt to provide a thorough documentation in the upcoming month. Please stay tuned and would value your feedback on the documentation part.
Thanks,
Swanand

**Comment by ignacy130** [comment]
I'm interested in this project as well, because I couldn't find any mongodb backup that suits my needs!

**Comment by sreepanchajanyam** [comment]
Is this project still active? If so I would like to use this in a production environment.
As of now, Tayra seems to be the only open source alternative to Ops Manager of MongoDB.

**Comment by viveksd87** [comment]
@ignacy130 , @sreepanchajanyam - sorry guys, this project is not in active development anymore.
Even if you plan to use it, it's compatible with only older versions of mongo, i.e 2.4 and lower.

**Comment by nneul** [comment]
Would be good if you updated the description on the project so that others don't have to dig through issues/etc. to find this out.

**Comment by kamronbatman** [comment]
@nneul I agree, and it would be nice if anyone has an alternative to link.  Thanks!

**Comment by nneul** [comment]
Best I've found so far is the ruby project:  https://github.com/journeyapps/mongo-oplog-backup
